.gitignore

```diff
@@ -1,11 +1,34 @@
 .ipynb_checkpoints
+.vscode/
+.DS_Store
+Manifest.toml
+/docs/build/
+
+# gfx
 *.png
 *.gif
 *.mp4
-*.dat
+*.svg
 *.pdf
+*.ogv
+
+# data file
+*.dat
+*.txt
+*.jld*
+
+# paraview
 *.vti
+*.vtr
+*.vts
+*.vtu
+*.vtp
+*.vtm
 *.pvd
-Manifest.toml
-/docs/build/
-.vscode/
+*.pvt*
+
+# on supercomputer
+*.log
+slurm*
```

Comment:
This looks like you are running cases in the same repo of the source code (maybe in `examples/`) and you are using the general `.gitignore` to ignore the files generated. While I think it is okay to add typical file extensions that surely won't be included in WaterLily, this system makes `.gitignore` an ever-growing bucket of different filenames. I would suggest having the WaterLily package in a directory, and an examples directory elsewhere. Then you can develop in the WaterLily directory normally, and in the other examples package mark WaterLily as a development package.

Reply:
Sure! this is a very good suggestion. I think I will just make .gitignore ordered then.